packages/core/scripts/setup/ios.ts

```diff
@@ -144,7 +144,8 @@ export const addUploadBuildPhase = {
             null,
             {
               shellPath: '/bin/sh',
-              shellScript: `EMBRACE_ID=${id} EMBRACE_TOKEN=${token} ${embRunScript}`,
+              shellScript: `export REACT_NATIVE_MAP_PATH="$CONFIGURATION_BUILD_DIR/main.jsbundle.map"
```

Comment:
is adding `export` needed here or are the env vars only relevant in the context of `embRunScript`?

Reply:
The variable is not being used in that scope, but inside run.sh. You need the export to be used inside the run.sh